lib/atomic-management.js

```diff
@@ -259,6 +261,9 @@ export default {
       atom.enablePersistence || configName || atom.config.mainSource
     );
     this.updateStatusBar();
+    //for test
+    savedConfigFile = path.join(atom.project.getPaths()[0], "/spec/test.cson")
+    CSON.writeFile(savedConfigFile, atom.config.get(), () => console.log("error"))
```

Comment:
Why is this necessary?

Reply:
This is for testing all the configuration settings. Currently, there is no API to get the current configuration in the project. This CSON write will write all the current configuration settings to a cson file and we will compare the cson file and config.cson for any difference.

Reply:
Won't this affect the user's local project directory though?

Reply:
It will  create a cson file called: "test.cson" in the spec directory. Is that a bad behavior?

Reply:
I think so...
@djh329 @josephch405 what do you think?